packages/linea-ccip-gateway/test/testVerifier.spec.ts

```diff
@@ -1,36 +1,36 @@
-import { Server } from "@chainlink/ccip-read-server";
-import { makeL2Gateway } from "../src";
-import { HardhatEthersProvider } from "@nomicfoundation/hardhat-ethers/internal/hardhat-ethers-provider";
-import { HardhatEthersHelpers } from "@nomicfoundation/hardhat-ethers/types";
-import { expect } from "chai";
+import { Server } from '@chainlink/ccip-read-server';
+import { makeL2Gateway } from '../src';
+import { HardhatEthersProvider } from '@nomicfoundation/hardhat-ethers/internal/hardhat-ethers-provider';
+import { HardhatEthersHelpers } from '@nomicfoundation/hardhat-ethers/types';
+import { expect } from 'chai';
 import {
   BrowserProvider,
   Contract,
+  ethers as ethersT,
```

Comment:
ethersT doesn't feel like a good name, I would have renamed the other one to ethersHardhat